fix: [CDS-95943]: Improves Documentation for Fetch Linked Apps (#6564)
harness/developer-hub: docs/continuous-delivery/gitops/pr-pipelines/pr-pipelines.md

5. Select **Submit**.
6. In the top-right corner, select **Save**.

### Specify Deployment Repo manifest details

> Once the Release Repo manifests are correctly configured, we add the Deployment Repo manifests in a similar manner. This specifies the path to your ApplicationSet template.

Reviewer: ```suggestion
Follow these steps once the Release Repo manifests are correctly configured to specify the path to your ApplicationSet template.
```

### Fetch Linked Apps step

The Fetch Linked Apps step provides app information, such as the app name, agent identifier, and URL to the Harness GitOps app. 

> Configuring the Deployment Repo manifest in your Service is required for this step to execute correctly.

Reviewer: ```suggestion
Configuring the Deployment Repo manifest in your service is required for this step to execute correctly.
```
